README.md

```diff
@@ -1 +1,149 @@
-# OOP_Libaray
\ No newline at end of file
+# OOP_Libaray
+<a name="readme-top"></a>
+ 
+
+<div align="center"> 
+  <img src="murple_logo.png" alt="logo" width="40"  height="auto" />
+  <br/>  
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Enumerable] <a name="about-project"></a>
+
+>This repo is a school library tool that allows you to record what books are in the library and who borrows them, the project focused on practicing the main four concepts in oop using ruby.
+
+**[your_project__name]**  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a> 
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features" /> 
+**[useing module]**
+**[useing all? any? filter?]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo" />
+
+- [Live Demo Link]()
+
+## 💻 Getting Started <a name="getting-started" />
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby
+- Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd "to where you want to install your project"
+
+ https://github.com/AhmadiGu/OOP_Libaray
+```
+
+### Install
+
+Install this project with:
+
+```
+ cd my-project
+  gem install
+```
+
+ 
+
+## 👥 Authors <a name="authors" /> 
+
+👤 **Gulalai Ahmadi**
+
+- GitHub: [GitHub Profile](https://github.com/AhmadiGu)
+- LinkedIn: [LinkedIn Profile](https://ahmadigu.github.io/Desktop_Portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[School libaray]**
```

Comment:
- Please, let's add some future features about the project as what you've added currently is not very descriptive. Please, let update this section of the readme to make it more professional.